go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/sirupsen/logrus v1.4.1
 	github.com/spf13/cobra v0.0.0-20160830174925-9c28e4bbd74e
 	github.com/spf13/viper v0.0.0-20150621231900-db7ff930a189
-	github.com/stellar/go v0.0.0-20230320165241-a5f3278b8282
+	github.com/stellar/go v0.0.0-20230327144237-ddcb0c7dea5a
```

Comment:
this duplicates PR [#535](https://github.com/stellar/soroban-tools/pull/535/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L17), so, 535 could be closed in favor of this pr instead correct?

Reply:
Yeah, I had already closed it.